app/assets/stylesheets/application.scss

```diff
@@ -96,3 +107,7 @@ h1 {
     z-index: -1000;
     overflow: hidden
 }
+
+main {
```

Comment:
Holy moly I take that back: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main